CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ x.x.x Release notes (yyyy-MM-dd)
 
 * Use reachability API to minimize the reconnection delay if the network
   connection was lost.
+* Add a new API for changing and retrieving permissions for synchronized Realms.
```

Comment:
I'd make this plural to indicate that we didn't just add a single method, but a family of types & methods:

```md
* Add new APIs for changing and retrieving permissions for synchronized Realms.
  These API are intended to replace the existing Realm Object-based permissions
  system.
```